auto-complete.js

```diff
@@ -54,33 +62,34 @@ var autoComplete = (function(){
             offsetTop: 1,
             cache: 1,
             menuClass: '',
-            renderItem: function (item, search, suggestionIndex){
+            renderItem: function (item, search, suggestionIndex) {
                 // escape special characters
                 search = search.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&');
                 var re = new RegExp("(" + search.split(' ').join('|') + ")", "gi");
                 return '<div class="autocomplete-suggestion" data-val="' + item + '">' + item.replace(re, "<b>$1</b>") + '</div>';
             },
-            onSelect: function(e, term, item){}
+            onSelect: function (e, term, item) { },
+            queryHistory: false
```

Comment:
This field receives a string, you shouldn't initialize it with a boolean value
```suggestion
            queryHistory: null
```

Reply:
Also, consider a more describable name like `queryHistoryStorageName`